src/app/App.tsx

```diff
@@ -36,8 +37,11 @@ function App() {
             .filter((country) =>
               country.name.toLowerCase().includes(searchText)
             )
-            .map((country) =>  
-              <Card country={country} />
+            .map((country) =>
+              <Link to={ `detail/${country.name.toLowerCase()}` }>
```

Comment:
we forget the key prop:
https://react.dev/learn/rendering-lists#keeping-list-items-in-order-with-key